rasa/core/policies/flow_policy.py

```diff
@@ -311,20 +312,20 @@ def get_value(
 
             return initial_value
 
-        text_slots = dict(
-            {
-                slot.name: get_value(tracker.get_slot(slot.name))
-                for slot in self.domain.slots
-            }
-        )
+        # attach context to the predicate evaluation to allow coditions using it
+        document: Dict[str, Any] = {
+            "context": FlowStack.from_tracker(tracker).current_context()
+        }
+        for slot in self.domain.slots:
+            document[slot.name] = get_value(tracker.get_slot(slot.name))
```

Comment:
we should probably also make sure that there is no slot with the name context

Reply:
yeah, but that should even happen globally - as it also creates issues for templates in nlg.

I did it in a backwards compatible way though, so if you have a context slot that will at least win.